Trace.py

```diff
@@ -8,14 +8,22 @@
 
 from helper import *
 
+OutputDir = "out"
```

Comment:
In the future we should get rid of these globals

Reply:
Yeah, it's already done, just another 5 stacked PRs to get to it :)